iOS Client/DevotionsController/DevotionsCoordinator.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DevotionsCoordinator.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 10/30/17.
+//  Copyright © 2017 Mockingbird. All rights reserved.
+//
+import Foundation
+import UIKit
+import ReSwift
+
+class DevotionsCoordinator: NSObject, Coordinator, StoreSubscriber {
+    var route: [RouteComponent] = [.base]
+    var tab: Tab = .devotions
+    var childCoordinators: [Coordinator] = []
+    var rootViewController: UIViewController {
+        return self.navigationController
+    }
+    
+    private lazy var navigationController: UINavigationController = {
+        return UINavigationController()
+    }()
+    
+    func start() {
+        let devotionsController = MBDevotionsViewController.instantiateFromStoryboard()
+        navigationController.pushViewController(devotionsController, animated: true)
+        MBStore.sharedStore.subscribe(self)
+        
+        MBStore().syncDevotions { devotions, error in
+            if error != nil {
+                MBStore.sharedStore.dispatch(LoadedDevotions(devotions: Loaded.error))
```

Comment:
Do these dispatches in a `DispatchQueue.main.async` block. I read on the ReSwift repo:

```
Note that the callback block from the network request arrives on a background thread, therefore we're using dispatch_async(dispatch_get_main_queue()) to perform the state update on the main thread. ReSwift will call reducers and subscribers on whatever thread you have dispatched an action from. We recommend to always dispatch from the main thread, but ReSwift does not enforce this recommendation. ReSwift will enforce that all Dispatches, Store Subscribes and Store Unsubscribes are on the same thread or serial Grand Central Dispatch queue. Therefore the main dispatch queue works, however the global dispatch queue, being concurrent, will fail.
```

Reply:
For now, it's fine b/c all the code in syncDevotions and getDevotionsWithCompletion is currently synchronous.